### PR TITLE
Dev into main

### PR DESCRIPTION
**Pull request by FGAT3**

### CODE REVIEW

File: client/src/components/ProfileInfos/ProfileInfos.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+function ProfileInfos({handleClickModal, showModalModify, setShowModalModify, showModalDelete, setShowModalDelete}) {
+  return (
+    <>
+    <h1>Profile</h1>
+      <img src={Avatar} alt="avatar par defaut" className="avatarImg" />
+      <h2>John Doe</h2>
+      <button
+        type="button"
+        onClick={handleClickModal(showModalModify, setShowModalModify)}
+        aria-label="Ouverture de la modale modifier mes informations"
```
Comment (Yoann-Fortin) [inline client/src/components/ProfileInfos/ProfileInfos.jsx]: Useless

File: client/src/components/ProfileInfos/ProfileInfos.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        className="buttonModify"
+      >
+        Modifier mes infos
+      </button>
+      <button
+        type="button"
+        onClick={handleClickModal(showModalDelete, setShowModalDelete)}
+        aria-label="Ouverture de la modale supprimer mes informations"
```
Comment (Yoann-Fortin) [inline client/src/components/ProfileInfos/ProfileInfos.jsx]: Useless

File: client/src/components/ScoreDisplay/ScoreDisplay.jsx
Comment (Yoann-Fortin) [inline client/src/components/ScoreDisplay/ScoreDisplay.jsx:1]: Isn't accessible

File: client/src/components/ScoreDisplay/ScoreDisplay.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@import "../../scss/index.scss";
+
+.tableScore {
+  color: $black-color;
+  width: 100%;
+
+  thead {
+    display: none;
```
Comment (Yoann-Fortin) [inline client/src/components/ScoreDisplay/ScoreDisplay.scss]: is removed on DOM to accessibility assistance. Privilegiate visibility hidden or another method to mask this

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by JadeJolivet — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**